plaso/filters/path_filter.py

```diff
@@ -593,24 +609,24 @@ def AddPathSegment(self, path_segment, scan_object):
 
     self._path_segments[path_segment] = scan_object
 
-  def GetScanObject(self, path_segment):
+  def GetScanObject(self, path_segment):  # pylint: disable=missing-return-type-doc
     """Retrieves the scan object for a specific path segment.
 
     Args:
-      path_segment: a string containing the path segment.
+      path_segment (str): path segment.
 
     Returns:
-      A scan object, which can be a scan tree sub node (instance of
-      PathFilterScanTreeNode), a path or the default value.
+      str|PathFilterScanTreeNode: a scan object, which is either
+          a scan tree sub node, a path of the default value.
```

Comment:
I don't really undersand. "a path to the default value" maybe?

Reply:
sry this should be "or" not "of"